tokens/usecase/pricing/chain/pricing_chain.go

```diff
@@ -221,8 +221,6 @@ func (c *chainPricing) computePrice(ctx context.Context, baseDenom string, quote
 	if useAlternativeMethod {
 		// Compute on-chain price for 1 unit of base denom and quote denom.
 		chainPrice = osmomath.NewBigDecFromBigInt(tenQuoteCoin.Amount.BigIntMut()).QuoMut(osmomath.NewBigDecFromBigInt(quote.GetAmountOut().BigIntMut()))
-	} else {
-		chainPrice = osmomath.OneBigDec().QuoMut(chainPrice)
```

Comment:
Note: This is a drive-by change. This was needed because we were confusing base and quote in `router_usecase.go`